docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+
+Infra/Circuit Breakers::
+* Fixed synchronizing inflight breaker with internal variable {es-pull}40878[#40878]
+
+Infra/Core::
+* Fail when using multiple data paths {es-pull}72184[#72184] (issue: {es-issue}71205[#71205])
+* Limit processors by available processors {es-pull}44894[#44894] (issue: {es-issue}44889[#44889])
+* Remove `nodes/0` folder prefix from data path {es-pull}42489[#42489]
+* Remove `bootstrap.system_call_filter` setting {es-pull}72848[#72848]
+* Remove `fixed_auto_queue_size` threadpool type {es-pull}52280[#52280]
+* Remove `node.max_local_storage_nodes` {es-pull}42428[#42428] (issue: {es-issue}42426[#42426])
+* Remove camel case named formats {es-pull}60044[#60044]
+* Remove legacy role settings {es-pull}71163[#71163] (issues: {es-issue}54998[#54998], {es-issue}66409[#66409], {es-issue}71143[#71143])
+* Remove `processors` setting {es-pull}45905[#45905] (issue: {es-issue}45855[#45855])
+* Remove the `local` parameter of `/_cat/nodes` {es-pull}50594[#50594] (issues: {es-issue}50088[#50088], {es-issue}50499[#50499])
+* Remove the listener thread pool {es-pull}53314[#53314] (issue: {es-issue}53049[#53049])
+* Remove the node local storage setting {es-pull}54381[#54381] (issue: {es-issue}54374[#54374])
+* Remove the `pidfile` setting {es-pull}45940[#45940] (issue: {es-issue}45938[#45938])
+* Removes `week_year` date format {es-pull}63384[#63384] (issue: {es-issue}60707[#60707])
+* Fail index creation using custom data path {es-pull}76792[#76792] (issue: {es-issue}73168[#73168])
+* System indices treated as restricted indices {es-pull}74212[#74212] (issue: {es-issue}69298[#69298])
+* Remove Joda dependency {es-pull}79007[#79007]
+* Remove Joda support from date formatters {es-pull}78990[#78990]
+* All system indices are hidden indices {es-pull}79512[#79512]
+
+Infra/Logging::
+* Remove slowlog level {es-pull}57591[#57591] (issue: {es-issue}56171[#56171])
+
+Infra/Plugins::
+* Remove deprecated basic license feature enablement settings from 8.0 {es-pull}56211[#56211] (issue: {es-issue}54745[#54745])
+
+Infra/REST API::
+* Remove content type required setting {es-pull}61043[#61043]
+* Remove deprecated endpoints containing `_xpack` {es-pull}48170[#48170] (issue: {es-issue}35958[#35958])
+* Remove deprecated endpoints of hot threads API {es-pull}55109[#55109] (issue: {es-issue}52640[#52640])
+* Allow parsing Content-Type and Accept headers with version {es-pull}61427[#61427]
+
+Infra/Resiliency::
+* Fail node containing ancient closed index {es-pull}44264[#44264] (issues: {es-issue}21830[#21830], {es-issue}41731[#41731], {es-issue}44230[#44230])
+
+Infra/Scripting::
+* Consolidate script parsing from object {es-pull}59507[#59507] (issue: {es-issue}59391[#59391])
+* Scripting: Move `script_cache` into _nodes/stats {es-pull}59265[#59265] (issues: {es-issue}50152[#50152], {es-issue}59262[#59262])
+* Scripting: Remove general cache settings {es-pull}59262[#59262] (issue: {es-issue}50152[#50152])
```

Comment:
```suggestion
* Move `script_cache` into _nodes/stats {es-pull}59265[#59265] (issues: {es-issue}50152[#50152], {es-issue}59262[#59262])
* Remove general cache settings {es-pull}59262[#59262] (issue: {es-issue}50152[#50152])
```

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c